front/app/components/Sharing/SharingDropdownButton/SharingDropdownContent.tsx

```diff
@@ -80,6 +86,14 @@ const EmailIcon = styled(Icon)`
   fill: ${colors.secondaryText};
 `;
 
+const CopyLinkIcon = styled(Icon)`
+  margin-right: 10px;
+  width: 22px;
+  height: 17px;
+  fill: ${colors.grey};
+  transform: rotateY(0deg) rotate(-45deg);
+`;
```

Comment:
Needed a styled icon here to apply the transform.

Reply:
I wonder if we shouldn't add this rotate property to the component library? Or perhaps we can go without it here?